public/video-ui/src/services/WorkflowApi.js

```diff
@@ -149,7 +149,21 @@ export default class WorkflowApi {
     });
   }
 
-  static updateStatus({id, status}) {
+  static updateProdOffice({ id, prodOffice }) {
+    const payload = {
+      data: prodOffice
+    };
+
+    return pandaReqwest({
```

Comment:
Typo? 😁

Reply:
(oh, wait, it's everywhere. NVM!)

Reply:
a kewl way of spelling I guess.. caught me out first time too